use_case_examples/hybrid_model/serve_model.py

```diff
@@ -0,0 +1,209 @@
+"""Hybrid Model Deployment Server.
+
+Routes:
+    - Get all names
+    - Get client.zip
+    - Add a key
+    - Compute
+"""
+
+import argparse
+import ast
+import io
+import time
+import uuid
+from collections import defaultdict
+from functools import lru_cache
+from pathlib import Path
+from typing import Optional, Tuple, Union
+
+import uvicorn
+from fastapi import FastAPI, Form, HTTPException, UploadFile
+from fastapi.responses import FileResponse, StreamingResponse
+from loguru import logger
+
+# No relative import here because when not used in the package itself
+from concrete.ml.deployment import FHEModelServer
+
+
+def underscore_str_to_tuple(tup):
+    return ast.literal_eval(tup.replace("p_", "(").replace("_p", ")").replace("_", ", "))
```

Comment:
This is the only blocking thing while trying to use the use case on phi1.5. Maybe something to improve here.

Reply:
The shape thing doesn't work properly?